### PR TITLE
Prepare provider readmes for 2020.11.07 version

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+### Release 2020.11.7
```
Comment (kaxil) [inline airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md:3]: Can you remind me again please on why we strip `0` in the date -- I can vaguely remember it had soemthing to do with PyPI release convention / parsing or stripping of 0??

**Review by turbaszek — APPROVED** [review]

**Comment by kaxil** [comment]
Can you please rebased your PR on latest Master since we have applied [Black](https://github.com/apache/airflow/commit/4e8f9cc8d02b29c325b8a5a76b4837671bdf5f68) and [PyUpgrade](https://github.com/apache/airflow/commit/8c42cf1b00c90f0d7f11b8a3a455381de8e003c5) on Master

File: airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md (hunk 1 above)
Comment (potiuk) [reply airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md:3]: Yep. That's it. PyPI won't accept 2020.11.07 as version.

**Comment by potiuk** [comment]
Closing this. I will open a new readme preparation with latest changes.

File: airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md (hunk 1 above)
Comment (potiuk) [reply airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md:3]: And we are releasing 11.13 eventually so no issue whatsoever :)

**Pull request closed without merging**